engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,29 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+Always maintain **an odd number of manager nodes** in the swarm. Otherwise, your
+swarm may lose a quorum of managers, and will not be able to perform management
+tasks. Three managers is the minimum, and five managers is recommended. In a
+swarm with 5 managers, you can lose 2 managers and still maintain a quorum.
+
+#### Recovering from losing a quorum
+
+If you lose a quorum of managers, you cannot administer the swarm. Attempting
+to remove a node from a swarm which does not have a quorum of managers results
+in an error like the following:
+
+```no-highlight
+Error response from daemon: rpc error: code = 4 desc = context deadline exceeded
+```
+
+The only way to recover from this state is to use the `--force-new-swarm` action
+from a manager node. This removes all managers except the manager the command
+was run from. Quorum is achieved because there is now only one manager. Promote
+nodes to be managers until you have the desired number of managers. You should
+always have an odd number of managers.
```

Comment:
I don't think we need to repeat the recommendation for an odd number of managers here. It's covered well under "Add manager nodes for fault tolerance".